GEM-data/eo-commercial-data.ipynb

```diff
@@ -0,0 +1,539 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Examples showcasing GEM EO commercial data collections"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is a notebook showcasing workflow examples for downloading EO data from commercial data collections available through Sentinel Hub.\n",
+    "\n",
+    "### Data download examples in this notebook:\n",
+    "\n",
+    "1.  PlanetScope \n",
+    "\n",
+    "2.  Pleiades\n",
+    "\n",
+    "3.  Spot\n",
+    "\n",
+    "4.  WorldView (+GeoEye)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import datetime\n",
+    "\n",
+    "import matplotlib.colors\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from aenum import MultiValueEnum\n",
+    "from eolearn.core import EOPatch, FeatureType\n",
+    "from eolearn.io import SentinelHubDemTask, SentinelHubEvalscriptTask, SentinelHubInputTask, get_available_timestamps\n",
+    "from matplotlib.colors import BoundaryNorm, ListedColormap\n",
+    "\n",
+    "from sentinelhub import CRS, Band, BBox, DataCollection, MimeType, SentinelHubRequest, SHConfig, Unit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### General parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "metadata": {
+    "pycharm": {
+     "name": "#%% \n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# regions of interest\n",
+    "\n",
+    "ps_bbox = BBox(bbox=[562218, 5174019, 564201, 5172501], crs=CRS.UTM_33N)\n",
+    "pleiades_bbox = wv_bbox = BBox(bbox=[562150.34, 5172481.37, 564214.37, 5174058.72], crs=CRS.UTM_33N)\n",
+    "spot_bbox = wv_bbox = BBox(bbox=[562150.34, 5172481.37, 564214.37, 5174058.72], crs=CRS.UTM_33N)\n",
+    "wv_bbox = BBox(bbox=[561924.43, 5172145.65, 564217.92, 5174392.08], crs=CRS.UTM_33N)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. PlanetScope ( [PS](https://docs.sentinel-hub.com/api/latest/data/planet/planet-scope/) )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 47,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ps_data = DataCollection.define_byoc(\n",
+    "    \"906b545b-4305-4968-b245-2abcb0609bc1\",\n",
+    "    service_url=\"https://services.sentinel-hub.com\",\n",
+    "    bands=[\n",
+    "        Band(name=\"Blue\", units=(Unit.DN,), output_types=(np.uint16)),\n",
+    "        Band(name=\"Green\", units=(Unit.DN,), output_types=(np.uint16)),\n",
+    "        Band(name=\"Red\", units=(Unit.DN,), output_types=(np.uint16)),\n",
+    "        Band(name=\"NIR\", units=(Unit.DN,), output_types=(np.uint16)),\n",
+    "    ],\n",
```

Comment:
Here you could also do
```python
band_names = ["Blue", "Green", "Red", "NIR"]
[ Band(name=name, units=(Unit.DN,), output_types=(np.uint16,)) for name in band_names)]
```
I also think you're missing a comma in `(np.uint16,)`.
This applies to other occurrences as well.